src/main/frontend/components/query/result.cljs

```diff
@@ -60,27 +60,28 @@
               (not (and (string? query) (string/includes? query "(by-page false)")))))))
 
 (defn get-query-result
+  "Fetches a query's result, transforms it as needed and saves the result into
```

Comment:
While writing this docstring I couldn't figure out why we still have the query-triggered? atom. Looks like it was last used in https://github.com/logseq/logseq/commit/b17c4ea37e17b396a088a99c445f6b9397b3cebf#r118810145. I could remove it tomorrow unless you beat me to it

Reply:
Agree to remove it. It's introduced to avoid running the same query && transforming multiple times, but the `query-result` cache seems buggy, so let's remove it for now. 
This time, I won't compete with you 😄